server/models/Category.js

```diff
@@ -13,8 +13,8 @@ const CategorySchema = new Schema({
         type: String
     },
     hideCategory: {
-        type: String,
-        default: 'off',
+        type: Boolean,
```

Comment:
call the field "hidden" or "visible" - so that we have consistent names for these fields across articles, categories and other collections (it's better to keep fields reused in most collections like hidden, dateAdded, dateUpdated etc... consistent)

Reply:
are you sure you want the default to be hidden? (up to you really - just want to make sure you thought it through)